platforms/documentation/docs/src/docs/userguide/authoring-builds/structuring/organizing_gradle_projects.adoc

```diff
@@ -0,0 +1,581 @@
+== Multi-Project Build (`include()`)
+
+image::structuring-builds-2.png[]
+
+In the settings file, you can use the `include` method to add another subproject to the root project:
+
+====
+include::sample[dir="snippets/multiproject/standardLayouts/kotlin",files="settings.gradle.kts[tags=hierarchical-layout]"]
+include::sample[dir="snippets/multiproject/standardLayouts/groovy",files="settings.gradle[tags=hierarchical-layout]"]
+====
+
+The `include` method takes <<intro_multi_project_builds#sec:project_path,project paths>> as arguments.
+The project path is assumed to be equal to the relative physical file system path.
+For example, a path `services:api` is mapped by default to a folder `./services/api` (relative to the project root `.`).
+
+More examples of how to work with the project path can be found in the DSL documentation of link:{groovyDslPath}++/org.gradle.api.initialization.Settings.html#org.gradle.api.initialization.Settings:include(java.lang.String[])++[Settings.include(java.lang.String[\])].
+
+Let's add another subproject called `lib` to the previously created project.
+
+All we need to do is add another `include` statement in the root settings file:
+
+====
+include::sample[dir="snippets/multiproject/multiple-subprojects/kotlin",files="settings.gradle.kts[]"]
+include::sample[dir="snippets/multiproject/multiple-subprojects/groovy",files="settings.gradle[]"]
+====
+
+Gradle will then look for the build file of the new `lib` subproject in the `./lib/` directory:
+
+[.multi-language-sample]
+=====
+[source, kotlin]
+----
+.       // <1>
+├── settings.gradle.kts // <2>
+├── app/                    // <3>
+│   ├── build.gradle.kts        // <4>
+│   └── src/
+└── lib/                    // <3>
+    ├── build.gradle.kts        // <4>
+    └── src/
+----
+=====
+[.multi-language-sample]
+=====
+[source, groovy]
+----
+.       // <1>
+├── settings.gradle // <2>
+├── app/                // <3>
+│   ├── build.gradle        // <4>
+│   └── src/
+└── lib/                // <3>
+    ├── build.gradle        // <4>
+    └── src/
+----
+=====
+<1> Root project
+<2> Settings file
+<3> Subproject
+<4> Subproject build file
+
+You can learn more about multi-project builds in <<multi_project_builds.adoc#multi_project_builds,Multi-Project Builds>>.
+
+[[sec:build_sources]]
+== Sharing Build Logic (`buildSrc`)
+
+image::structuring-builds-3.png[]
+
+When projects grow in size and complexity, it's common to see the same logic repeated across multiple subprojects—like applying the same plugins, configuring the same tasks, or declaring the same dependencies.
+
+Duplicated build logic is hard to maintain and easy to get wrong.
+Gradle provides a built-in way to centralize and reuse this logic: a special directory called `buildSrc`.
+
+`buildSrc` is a convention-based embedded build located in the root of your Gradle project.
+Any code you put in this directory is automatically compiled and added to the classpath of your main build.
```

Comment:
I'm wondering if "Any code" is the best wording we can use here, versus "Any build logic"? Could we check with someone in engineering on which one is more correct?

Reply:
Updated based on Tom's feedback.